tools/src/main/java/org/apache/kafka/trogdor/workload/RoundTripWorker.java

```diff
@@ -332,6 +340,7 @@ public void run() {
                 long lastLogTimeMs = Time.SYSTEM.milliseconds();
                 while (true) {
                     try {
+                        lock.lock();
```

Comment:
It's definitely overkill to lock the whole function body, right?  I guess we only have one thread running this now, but it could be an issue if we want to have multiple threads run it in the future.

Let's just replace the `AtomicLong` with a regular long, and use the lock to protect changes to the `unackedSends` variable.